### PR TITLE
[xar] Add missing options

### PR DESCRIPTION
**Pull request by ffflorian**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/finnp/xar/blob/master/readme.md
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/xar/index.d.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
 // Type definitions for xar 1.1
 // Project: https://github.com/finnp/xar
-// Definitions by: Florian Keller <https://github.com/ffflorian>
+// Definitions by: Florian Imdahl <https://github.com/ffflorian>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node"/>
 
-import { Readable as ReadableStream } from 'stream';
+import { Readable as ReadableStream } from 'readable-stream';
```
Comment (peterblazejewicz) [inline types/xar/index.d.ts:8]: where is that direct dependency defined in th esource package? what is the gain here? The package (xar) was updated 5 years ago

**Comment by elibarzilay** [comment]
@ffflorian: what's the status of this?

File: types/xar/index.d.ts (hunk 1 above)
Comment (ffflorian) [reply types/xar/index.d.ts:8]: You are right, using `stream` is enough, although `readable-stream` is a core module from Node.js which doesn't need to be defined in the source.

Comment (ffflorian) [reply types/xar/index.d.ts]: I updated my PR.

**Comment by ffflorian** [comment]
@elibarzilay I addressed the PR review.

**Review by peterblazejewicz — APPROVED** [review]

**Comment by ffflorian** [comment]
Thanks for the review @peterblazejewicz!
Ready to merge

**Comment by ffflorian** [comment]
Ready to merge

**Pull request merged**